Representable/representable
# Rename Campaign to Drive, add dashboard tests

First off, sorry for the number of files changed in this PR! The bulk of file changes happened because I replaced all the instances of the word campaign. The second part is that I created some overdue tests for the dashboard because I wanted to get in the habit of adding tests for each PR. @jessiejofielding - these tests may be helpful for the multi community export feature.

Key Changes:
- Renames Campaign model -> Drive 
- Replaces all references to campaign with drive
- Adds basic tests for dashboard w/ fixture

Tests added:
- Test for creating organization with view (sends post request,  verifies successful response)
- Test for the dashboard home page (creates an organization, then retrieves home_org template, verifies successful response)
- Test for the absolute url of an organization (retrieves template, verifies successful response)
- Test for the dashboard drive page (retrieves template, verifies successful response)
- Test for the org review page (retrieves template, verifies successful response)
- Test for the dashboard drive review page (retrieves template, verifies successful response)

To test this PR:
- Run migrations
- Check out `test_organization.py`: check that this is what you expect to be testing
- Check out fixtures: `single_org.json`: notice that this fixture creates exactly one user who is an admin of an organization, one organization, and one drive.
- Run `python manage.py test`
- Click through dashboard pages for any errors
- you may want to try creating an entry for a drive just to double check the links weren't broken

Screenshots:
<img width="1137" alt="Screen Shot 2020-07-03 at 5 08 08 PM" src="https://user-images.githubusercontent.com/8892509/86501864-ededb280-bd51-11ea-8b88-b48563016932.png">
<img width="1142" alt="Screen Shot 2020-07-03 at 5 07 58 PM" src="https://user-images.githubusercontent.com/8892509/86501867-f0e8a300-bd51-11ea-9bae-5c9658eafbd5.png">


resolves #249
